csharp/protos/extensions/extensions_a.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
```

Comment:
why put these files under a new "extensions" directory? they'd be fine in csharp/protos just as the other .proto files for unittests?